packages/mobile/src/identity/verification.ts

```diff
@@ -320,16 +354,21 @@ export function* doVerificationFlow(withoutRevealing: boolean = false) {
 
       yield put(setVerificationStatus(VerificationStatus.CompletingAttestations))
       yield race({
-        actionableAttestationCompleted: all([
-          call(completeAttestations, attestationsWrapper, account, phoneHashDetails, attestations),
-          // Set acccount and data encryption key in Accounts contract
-          // This is done in other places too, intentionally keeping it for more coverage
-          call(registerAccountDek, account),
-        ]),
+        actionableAttestationCompleted: call(
+          completeAttestations,
+          attestationsWrapper,
+          account,
+          phoneHashDetails,
+          attestations
+        ),
         // This is needed, because we can have more actionableAttestations than NUM_ATTESTATIONS_REQUIRED
         requiredAttestationsCompleted: call(requiredAttestationsCompleted),
       })
 
+      // Set acccount and data encryption key in Accounts contract
+      // This is done in other places too, intentionally keeping it for more coverage
+      yield spawn(registerAccountDek, account)
```

Comment:
why move `registerAccountDek` out of `race` above?

Reply:
IIUC we want DEK registration no matter what happens first. Previously DEK registration would only be happening if `actionableAttestationCompleted` won the race but not if `requiredAttestationsCompleted` was first. This way, DEK registration is attempted no matter what.

Reply:
that make sense, what do you think of the following then: `all(registerAccountDek, race(actionableAttestationCompleted, requiredAttestationsCompleted))`. This way we run `registerAccountDek` in parallel and guarantee that it was called. Timewise it should not be too different, but we avoid running detach task.

Reply:
Sounds good to me, will do